extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,540 @@
+
+A **buffer** is a binary blob. A single buffer can be stored within the binary chunk of a subtree file. Further buffers can be stored as individual binary files that are referred to by the `buffer.uri` property. The buffers can store the availability data of a subtree in binary form, or other data that is associated with a subtree, like metadata for implicit tiles defined using the [`3DTILES_metadata` extension.](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/extensions/3DTILES_metadata#implicit-tile-metadata)
+
+Each buffer has a `byteLength` describing the size of the data, including any padding (for subtree binary files). 
+
+A **buffer view** is a contiguous subset of a buffer. A buffer view's `buffer` property is an integer index to identify the buffer. A buffer view has a `byteOffset` and a `byteLength` to describe the range of bytes within the buffer. The `byteLength` does not include any padding. There may be multiple buffer views referencing a single buffer.
+
+For efficient memory access, the `byteOffset` of a buffer view must be aligned to a multiple of 8 bytes.
+
+> **Example:** The JSON description of a subtree where each tile is available, but not all tiles have content, and not all child subtrees are available:
+> 
+> ```json
+> {
+>   "buffers": [
+>     {
+>       "name": "Internal Buffer",
+>       "byteLength": 16
+>     },
+>     {
+>       "name": "External Buffer",
+>       "uri": "external.bin",
+>       "byteLength": 32
+>     }
+>   ],
+>   "bufferViews": [
+>     {
+>       "buffer": 0,
+>       "byteOffset": 0,
+>       "byteLength": 11
+>     },
+>     {
+>       "buffer": 1,
+>       "byteOffset": 0,
+>       "byteLength": 32
+>     }
+>   ],
+>   "tileAvailability": {
+>     "constant": 1,
+>   },
+>   "contentAvailability": {
+>     "bufferView": 0
+>   },
+>   "childSubtreeAvailability": {
+>     "bufferView": 1
+>   }
+> }
+> ```
+
+When all tiles exist, then their availability can be encoded by setting `tileAvailability.constant` to `1`, without needing an explicit bitstream.
+
+Only some tiles have content, and `contentAvailability.bufferView` indicates where the bitstream for the content availability is stored: The `bufferView` with index 0 refers to the `buffer` with index 0. This buffer does not have a `uri` property, and therefore refers to the _internal_ buffer. The `byteOffset` and `byteLength` indicate that the content availability bitstream is stored in the bytes `[0...11)` of the internal buffer.
+
+Some child subtrees exist, so `childSubtreeAvailability.bufferView` refers to another bitstream. The `bufferView` with index 1 refers to the buffer with index `1`. This buffer has a `uri` property, indicating that this second bitstream is stored in an external binary file.
+
+### Availability Packing
+
+Availability bitstreams are packed in binary using the format described in the [Booleans](../../specification/Metadata#booleans) section of the Cesium 3D Metadata Specification.
+
```

Comment:
Not sure where to put this (Maybe make a new examples section here?) but I got feedback from Patrick that we want to show another use case: making a global tileset. Typically we do this with a "double-headed quadtree" (one quadtree for the western hemisphere and one for the eastern hemisphere). That would look like this:

```
{
  "asset": {
    "version": "1.0"
  },
  "geometricError": 8000,
  "extensionsUsed": [
    "3DTILES_implicit_tiling"
  ],
  "extensionsRequired": [
    "3DTILES_implicit_tiling"
  ],
  "root": {
    "boundingVolume": {
      "region": [-3.14159265359, -1.57079632679, 3.14159265359, 1.57079632679, 0, 1000]
    },
    "refine": "REPLACE",
    "geometricError": 4000,
    "children": [
      {
        "boundingVolume": {
          "region": [-3.14159265359, -1.57079632679, 0, 1.57079632679, 0, 1000]
        },
        "content": {
          "uri": "content/western-hemisphere/{level}/{x}/{y}.b3dm"
        },
        "geometricError": 2000,
        "extensions": {
          "3DTILES_implicit_tiling": {
            "subdivisionScheme": "QUADTREE",
            "maximumLevel": 20,
            "subtrees": {
              "uri": "subtrees/eastern-hemisphere/{level}/{x}/{y}.subtree"
            },
            "subtreeLevels": 7
          }
        }
      },
      {
        "boundingVolume": {
          "region": [0, -1.57079632679, 3.14159265359, 1.57079632679, 0, 1000]
        },
        "content": {
          "uri": "content/eastern-hemisphere/{level}/{x}/{y}.b3dm"
        },
        "geometricError": 2000,
        "extensions": {
          "3DTILES_implicit_tiling": {
            "subdivisionScheme": "QUADTREE",
            "maximumLevel": 20,
            "subtrees": {
              "uri": "subtrees/eastern-hemisphere/{level}/{x}/{y}.subtree"
            },
            "subtreeLevels": 7
          }
        }
      }
    ]
  }
}

```

Reply:
Note entirely sure what to do with this one. In `content-gltf`, there was an example, and Sean suggested to remove it, because the examples should be in `3d-tiles-samples` in the future (and then, we could add a link here). 

Maybe we'd defer this until we actually _have_ examples? 

(I mean, real examples. The ones that you linked to in https://github.com/CesiumGS/3d-tiles/issues/512#issuecomment-952284192 are  a bit "trivial" - _none_ of them has more than one subtree layer...)

Reply:
okay fair, I'll open an issue for this.

Reply:
I left a comment here: https://github.com/CesiumGS/3d-tiles/issues/512#issuecomment-956636721